web-gateway/app/controllers/Main.java

```diff
@@ -17,6 +17,8 @@
     private final UserService userService;
     private final FormFactory formFactory;
 
+    public static boolean showInlineInstruction = true;
```

Comment:
Hi @mehulsr, rather than using a `public static` variable, prefer using a property we can setup on `application.conf` and provide a default value via a `reference.conf`. Then, in the code, we can read the value using the `Configuration` instance (we can get it injected into each `Controller`).

I'll let you work out how to pass the value from the `Controller` to the view. ;-)